<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>621</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Actions</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Température</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajout de pH</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>60</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Lumière</string>
     </property>
    </widget>
    <widget class="QSlider" name="temperature_slider">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>160</width>
       <height>19</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_temp">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>61</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_ph">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>92</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>On</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_light">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>60</y>
       <width>92</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>On</string>
     </property>
    </widget>
   </widget>
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>29</y>
      <width>531</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAccueil">
    <property name="title">
     <string>Accueil</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuLive">
    <property name="title">
     <string>Live</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHistorique_Temp_rature">
    <property name="title">
     <string>Historique Température</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHistorique_pH">
    <property name="title">
     <string>Historique pH</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuParam_tres">
    <property name="title">
     <string>Paramètres</string>
    </property>
   </widget>
   <addaction name="menuAccueil"/>
   <addaction name="menuLive"/>
   <addaction name="menuHistorique_Temp_rature"/>
   <addaction name="menuHistorique_pH"/>
   <addaction name="menuParam_tres"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
